articles/aws-vs-google-cloud/index.md

```diff
@@ -0,0 +1,106 @@
+
+Amazon Web Services (AWS) and Google Cloud are two cloud computing giants that provide cloud computing services. AWS is a platform owned by Amazon, while Google Cloud platform is owned by Google. These platforms compete heavily to gain a big market share in cloud computing.
+
+This article provides a comprehensive comparison between these cloud computing platforms. The comparison will involve analyzing their differences using key parameters. It will also highlight their advantages and disadvantages. By the end of the article, readers will have a good overview of these platforms, which will enable them to make an informed choice. 
+
+### What is AWS?
+Amazon Web Services (AWS) provides APIs and a cloud computing platform to individuals, companies, and governments. The AWS platform offers cloud computing solutions that are flexible, easy-to-use, scalable, reliable, and cost-effective. 
+
+It also consists of offerings such as infrastructure as a service (IAAS), software as a service (SAAS), and platform as a service (PAAS). Examples of the services provided by AWS include networking, storage, remote computing, mobile development, email, analytics, and security.
+
+![Amazon Web Services](/engineering-education/aws-vs-google-cloud-services/amazon-web-services.png)
+
+[Image Source: L2L International](https://www.l2linternational.com/wp-content/uploads/2018/08/amazonaws.png)
+
+### What is Google Cloud?
+Google Cloud is a cloud computing platform that provides cloud services such as data storage, machine learning, and data analytics. It also provides offerings such as serverless computing, platform as a service, and infrastructure as a service. 
+
+Other tools offered by this platform include networking, big data, the internet of things (IoT), management tools, security, and Cloud AI.  Google cloud services share the same infrastructure with other end-user products of Google. These include YouTube, Gmail, and Google Search.
+
+![Google Cloud Platform](/engineering-education/aws-vs-google-cloud-services/google-cloud-platform.jpg)
+
+[Image Source: Software Logic](http://softwarelogic.net/wp-content/uploads/2017/07/google-cloud1.jpg)
+
+### Features of AWS
+* They have a low cost of ownership compared to dedicated or private servers.
+* The AWS platform has hybrid capabilities that give it a cutting edge over its competitors.
+* Billing and management in the AWS platform are centralized.
+* In AWS, applications can be deployed in multiple regions around the globe.
+* AWS has an introduction feature called Identity and Access Management (IAM). This feature enhances the creation of groups or users. 
+* It enhances virtual privacy through the provision of a Virtual Private Cloud (VPC) that does not interfere with other networks. 
+
+### Features of Google Cloud
+* It allows the constant inclusion of more languages.
+* In Google Cloud platforms, user experience can be improved if a better user interface is used.

```

Comment:
I don't think this can be considered a feature

Reply:
I have deleted and replaced it with another good feature.